api/src/main/java/org/openmrs/module/ModuleFactory.java

```diff
@@ -1169,10 +1169,7 @@ public static void stopModule(Module mod, boolean isShuttingDown) {
 						String extId = ext.getExtensionId();
 						try {
 							List<Extension> tmpExtensions = getExtensions(extId);
-							if (tmpExtensions == null) {
-								tmpExtensions = new Vector<Extension>();
-							}
-							
+							tmpExtensions = new Vector<Extension>();
```

Comment:
Oh, I just realized that. The method never returns null though, it does the null checking itself.